### DISCUSSION TITLE
Feature Request: Notification on Fail

### DISCUSSION THREAD

**Discussion by ogezue** [teslamate-org/teslamate#1426] [Ideas]
Not for the first time I had a week where TeslaMate did not collect any data. This time because of MFA asking for a token and I did not recognised this. 

Wouldn't it be great if the user would be notified by mail that the service needs some help?

**Comment by eliluong** [comment]
If you can code, you can use the available MQTT topics to notify you when there's an issue. `teslamate/cars/$car_id/healthy` could be one topic, or you can come up with some other algorithm. Then you can send notification to email/Slack/IFTTT/etc. That's as close as you can get for now.

**Comment by scissorfighter** [comment]
I second this request!
